src/main/java/au/gov/nla/dlir/models/event/Correction.java

```diff
@@ -0,0 +1,15 @@
+package au.gov.nla.dlir.models.event;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class Correction {
+  private String type;
+  private Integer lineNumber;
+  private String correctedText;
```

Comment:
Should this be a List to cater for multiple new lines?

Reply:
Wondering if we need to treat each new line addition as a separate correction. This way the updating correction history would be cleaner.

Reply:
It shouldn't be a problem. Btw we are passing the entire dom for newspapers atm should be much less overhead compared to that.